src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/rewriter/subquery/utils/NestedQueryDetectorTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.unittest.rewriter.subquery.utils;
+
+import com.alibaba.druid.sql.ast.expr.SQLIdentifierExpr;
+import com.alibaba.druid.sql.ast.statement.SQLExprTableSource;
+import com.alibaba.druid.sql.ast.statement.SQLJoinTableSource;
+import com.alibaba.druid.sql.ast.statement.SQLJoinTableSource.JoinType;
+import com.amazon.opendistroforelasticsearch.sql.rewriter.subquery.NestedQueryContext;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
Should we avoid using *? I remember a comment along those lines on one of my CRs

Reply:
@davidcui-amzn  If you make the current checkstyle rules applicable to test folder as well, then the build won't succeed until all of the errors are fixed, including the * import. It's a little tedious work, but it's only a one-time thing, and will save a lot of comments in the future.

Reply:
Done.